art/src/node.rs

```diff
@@ -6,6 +6,11 @@ use core::ops::{Deref, DerefMut};
 use crossbeam_utils::CachePadded;
 use either::Either;
 
+//use macro
```

Comment:
What's the meaning of this comment?

Reply:
in rust 2018 version, 
you can use macro by `use` 
this is comment for that

Reply:
Ah, I see.  In that case, I prefer to remove the comments, because it's not about this crate's logic.